### PR TITLE
Shows number of actions which are pending moderation.

### PR DESCRIPTION
**Pull request by steelstyle**
Fixes CaptainFact/captain-fact#95.

### CODE REVIEW

**Comment by steelstyle** [comment]
@Betree any concern on the approach / refresh rate ?

File: app/API/graphql_queries.js [hunk 1]
```diff
@@ -72,3 +72,11 @@ export const loggedInUserUnreadNotificationsCount = gql`
     }
   }
 `
+
+export const loggedInUserPendingModerationCount = gql`
+  query LoggedInUserUnreadNotificationsCount {
+    loggedInUser {
+      actions_pending_moderation
```
Comment (Betree) [inline app/API/graphql_queries.js:79]: For Apollo's caching, it's good to always fetch the `id` (even when not using it)

```suggestion
      id
      actions_pending_moderation
```

**Review by Betree — COMMENTED** [review]
It's looking good!

My only concern is that, while loading, we show `0`. If `get(data, 'loggedInUser.actions_pending_moderation')` is undefined, we should not show anything. But this one can be part of a later PR if you don't have the time to iterate on it.

**Comment by steelstyle** [comment]
I saw that you took care of hiding the badge instead of showing 0, great !
Also I will keep that in mind for the future.

**Comment by Betree** [comment]
Indeed! Sorry for not pinging you about the change, I did it a bit in a hurry ^^

**Comment by steelstyle** [comment]
NP :) I saw this before considering working on that !

**Pull request merged**